src/System.Runtime/tests/System/SByteTests.cs

```diff
@@ -171,9 +171,11 @@ public static void ToString(sbyte i, string format, IFormatProvider provider, st
         [Fact]
         public static void ToString_InvalidFormat_ThrowsFormatException()
         {
-            IComparable comparable = (sbyte)123;
-            AssertExtensions.Throws<ArgumentException>(null, () => comparable.CompareTo("a")); // Obj is not a sbyte
```

Comment:
Why are we removing these tests?

Reply:
Accident, these were meant to be moved to a `CompareTo_ObjectNotByte_ThrowsArgumentException` method.

Reply:
Nevermind. These are already covered by the above method.

Reply:
See https://github.com/dotnet/corefx/pull/35567/files/2d822f19dab2be7c224cbb8a16a96b33b453cfe0#diff-a594bae1c8fdb7c47174dc16063e58feL65